Create detra.html

codingpixi opened the pull request:
Week 1 Homework Assignment

codingpixi committed:
Create detra.html

Week 1 Homework Assignment

codingpixi committed:
Update detra.html

nellarro commented on students/detra.html:
```diff
@@ -0,0 +1,51 @@
+    <title>Detra</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <style>
+        body {
```
💭 Would love to see this with a CSS style sheet!

codingpixi replied:
Oh sure thing. I wasn't really sure I could but I definitely will do that and add more css. Thx

nellarro replied:
You don't have to update it with CSS 😊. The assignment doesn't ask for a stylesheet. But, if you could link your html to the main html, that would be lovely! 🤗

nellarro reviewed (changes requested):
❓ Doesn't seem like you've linked your awesome work to the main `index.html` file. I want to be able to click on your name and get to your story! Once you link it up, should be good to go!

codingpixi committed:
Add files via upload

This is an updated version of my homework 1 assignment with more CSS

codingpixi commented:
I uploaded my new files, but I want to verify that you all see them. Thx

nselikoff commented:
Hi @codingpixi, I see the new files but if we were to merge this PR as-is, it would replace the site's main `index.html` page with your `index.html` page. My recommendation would be to revert to the previous commit (your original `Detra.html` file) and add the link from the site's main `index.html` page to your page per @nellarro's comment. Alternatively, if you want to use the new files, put them in a subdirectory under `students`, so you end up with an overall file structure like this:

```
index.html (main index file)
students
  Detra
    index.html
    resources/css/style.css
    (other files in resources dir)
    vendors/css/grid.css
    (other files in vendors dir)
```

codingpixi commented:
Thanks Nathan.

I want to make sure I am understanding you correctly. GitHub is a weakness I’d love to turn into a strength. If I upload my files to (https://github.com/codingpixi/int-web-dev/tree/master/students) this would allow me to keep the files that I have by using the subdirectory? Or do I need to add a folder titled Detra and then upload the files inside the Detra folder.

Detra
On Aug 9, 2020, 9:55 PM -0400, Nathan Selikoff <notifications@github.com>, wrote:
> Hi @codingpixi, I see the new files but if we were to merge this PR as-is, it would replace the site's main index.html page with your index.html page. My recommendation would be to revert to the previous commit (your original Detra.html file) and add the link from the site's main index.html page to your page per @nellarro's comment. Alternatively, if you want to use the new files, put them in a subdirectory under students, so you end up with an overall file structure like this:
> index.html (main index file)
> students
>  Detra
>    index.html
>    resources/css/style.css
>    (other files in resources dir)
>    vendors/css/grid.css
>    (other files in vendors dir)
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

codingpixi committed:
Add files via upload

From Nathan's comments to me I believe this is where I should've uploaded my files in order to not mess up Jenell's setup. Please let me know if I need to do anything differently. I did email Nathan last night but haven't heard back so I wanted to go ahead and do this. Thx

nselikoff commented:
@codingpixi don't worry about it for now, we'll chat more about git and GitHub tonight. If you're using the "Upload files" button on github.com, that's actually going to make it more confusing, so we'll get you either on GitHub Desktop or using the terminal to interact with the repository.

codingpixi commented:
Hi Nathan -

I believe I’ve connected my files to GitHub and created a branch titled detra. I am trying to work on my wk2 homework but need a bit more information from here. I know I need to get my files in Jenell’s student folder. Could you please point me in the right direction to get this done so I can proceed? Thx
On Aug 10, 2020, 5:29 PM -0400, Nathan Selikoff <notifications@github.com>, wrote:
> @codingpixi don't worry about it for now, we'll chat more about git and GitHub tonight. If you're using the "Upload files" button on github.com, that's actually going to make it more confusing, so we'll get you either on GitHub Desktop or using the terminal to interact with the repository.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

codingpixi commented:
Should I close this pull request? I cloned the this repo via terminal and have mad a new folder in the student file titled detra which includes a html and css file. I am trying to set it up to push my changes, but don't am getting a 403 error so there is a step or two that I'm missing, but don't know if this is holding it up

nselikoff commented:
Hi Detra, for the week 1 assignment, the workflow was to fork from Jenell's repo, make your changes in your fork in a branch besides master, and create a pull request to merge your changes back into Jenell's repo (this pull request). Please do continue to work on this pull request as you have time, so we can get it merged and have your page incorporated into the main class website.

For the week 2 assignment, you're starting with a brand new repo of your own (which I can see you already created), and you won't be creating any pull requests to Jenell because you're not merging into an upstream repo. The end goal is to deploy your image gallery to the web via surge.sh.

I think you may have the two repos confused, because I see a `detra` branch on the image-gallery repo, but not on the int-web-dev repo. 

Stick with it! Git and GitHub are confusing at first. In addition to the [GitHub resources in Google Classroom](https://classroom.google.com/u/0/c/MTQxMzczOTcyOTY4/m/MTQxNjU4NzM5ODcy/details), you may want to google some tutorials for Git, GitHub and the IDE you're using (e.g. VS Code, Brackets, Atom, Sublime, etc) to see how they work together.

codingpixi commented:
GM Nathan -

That’s this issue I was having, I cloned the nellarro/int-web-dev folder to my local machine. Created a detra folder that contains my html and css files. But when pushing it back I’m getting the errors. I’m including a screenshot of it.

Thx for being patient and responding to my questions. I truly appreciate it.
On Aug 12, 2020, 10:15 AM -0400, Nathan Selikoff <notifications@github.com>, wrote:
> Hi Detra, for the week 1 assignment, the workflow was to fork from Jenell's repo, make your changes in your fork in a branch besides master, and create a pull request to merge your changes back into Jenell's repo (this pull request). Please do continue to work on this pull request as you have time, so we can get it merged and have your page incorporated into the main class website.
> For the week 2 assignment, you're starting with a brand new repo of your own (which I can see you already created), and you won't be creating any pull requests to Jenell because you're not merging into an upstream repo. The end goal is to deploy your image gallery to the web via surge.sh.
> I think you may have the two repos confused, because I see a detra branch on the image-gallery repo, but not on the int-web-dev repo.
> Stick with it! Git and GitHub are confusing at first. In addition to the GitHub resources in Google Classroom, you may want to google some tutorials for Git, GitHub and the IDE you're using (e.g. VS Code, Brackets, Atom, Sublime, etc) to see how they work together.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

nselikoff commented:
Hi @codingpixi, you need to fork `nellarro/int-web-dev` first, then clone from your fork, which will be `codingpixi/int-web-dev`. Then create the new branch, commit, and push - this will push to your fork. Then you create a PR from the GitHub website.

I don't see a screenshot btw.
